packages/components/src/components/navigation-item/navigation-item.test.ts

```diff
@@ -295,4 +296,29 @@ describe('<sd-navigation-item>', () => {
       });
     });
   });
+
+  // Test separate variant
+  describe('when given a value for the "children" slot, an "href" property and "separate" property is true', () => {
+    it('renders a div with slot closed and a link in a separate div', async () => {
+      const el = await fixture<SdNavigationItem>(
+        html`<sd-navigation-item href="#" separate>${defaultSlot}${childrenSlot}</sd-navigation-item>`
```

Comment:
Optimisation: I feel like separate is not very speaking, EVEN WITH knowledge of the component. I do not know what is separated from what.

Reply:
We had quite some discussion on the wording and this was the final conclusion we had after a round with 3 or 4 people. 😄

We're currently rewriting our docs in a different branch, where this will be nicely communicated to the users of the library.

From a grammar perspective: We try to stick to describing the state of a component as in default HTML elements `disabled`, `checked` etc. instead of describing an action like `separate` or `split-actions` or similar.